<launch>

    <node pkg="control" exec="wpt_tracker_node" name="control_node" output="screen">
        <param name="control_frequency" value="20"/>
        <remap from="control/wpt_tracker/control_command" to="control/control_command"/>
    </node>

</launch>
